themes/dark/assets/js/config/search.js

```diff
@@ -79,6 +79,8 @@ MEDUSA.config.search = function() {
 
             if (selectedProvider.toLowerCase() === 'utorrent') {
                 client = 'uTorrent';
+                $('#label_warning_utorrent').show();
+                $('#label_anime_warning_utorrent').show();
```

Comment:
The slim, light and dark theme should all be using the same code.

Reply:
Done.